lib/vue/src/components/map/CMap.vue

```diff
@@ -62,9 +62,22 @@
         :key="`marker-${i}`"
         :lat-lng="marker.value"
         :icon="getIcon(marker.color)"
```

Comment:
Im thinking that if you style the map to always expand to the parent div, you'll have a much easier time styling it for each use case. Might be worth a few minutes of trying it